libs/designer/src/lib/ui/panel/panelroot.tsx

```diff
@@ -305,43 +325,54 @@ export const PanelRoot = (props: PanelRootProps): JSX.Element => {
-      panelScope={PanelScope.CardLevel}
-      panelHeaderControlType={getPanelHeaderControlType() ? PanelHeaderControlType.DISMISS_BUTTON : PanelHeaderControlType.MENU}
-      panelHeaderMenu={getPanelHeaderMenu()}
-      selectedTab={selectedPanelTab}
-      showCommentBox={showCommentBox}
-      tabs={registeredTabs}
-      nodeId={selectedNode}
-      onDismissButtonClicked={handleDelete}
-      readOnlyMode={readOnly}
-      setSelectedTab={setSelectedTab}
-      toggleCollapse={() => {
-        //only run validation when collapsing the panel
-        if (!collapsed) {
-          Object.keys(inputs?.parameterGroups ?? {}).forEach((parameterGroup) => {
-            inputs.parameterGroups[parameterGroup].parameters.forEach((parameter) => {
-              const validationErrors = validateParameter(parameter, parameter.value);
-              dispatch(
-                updateParameterValidation({ nodeId: selectedNode, groupId: parameterGroup, parameterId: parameter.id, validationErrors })
-              );
+    <>
+      <PanelContainer
+        {...commonPanelProps}
+        cardIcon={iconUri}
+        comment={comment}
+        noNodeSelected={!selectedNode}
+        isError={errorInfo?.level === ErrorLevel.Critical || opQuery?.isError}
+        errorMessage={errorInfo?.message}
+        isLoading={isLoading}
+        panelScope={PanelScope.CardLevel}
+        panelHeaderControlType={getPanelHeaderControlType() ? PanelHeaderControlType.DISMISS_BUTTON : PanelHeaderControlType.MENU}
+        panelHeaderMenu={getPanelHeaderMenu()}
+        selectedTab={selectedPanelTab}
+        showCommentBox={showCommentBox}
+        tabs={registeredTabs}
+        nodeId={selectedNode}
+        onDismissButtonClicked={handleDelete}
+        readOnlyMode={readOnly}
+        setSelectedTab={setSelectedTab}
+        toggleCollapse={() => {
+          //only run validation when collapsing the panel
+          if (!collapsed) {
+            Object.keys(inputs?.parameterGroups ?? {}).forEach((parameterGroup) => {
+              inputs.parameterGroups[parameterGroup].parameters.forEach((parameter) => {
+                const validationErrors = validateParameter(parameter, parameter.value);
+                dispatch(
+                  updateParameterValidation({ nodeId: selectedNode, groupId: parameterGroup, parameterId: parameter.id, validationErrors })
+                );
+              });
             });
-          });
-        }
-        togglePanel();
-      }}
-      trackEvent={handleTrackEvent}
-      onCommentChange={onCommentChange}
-      title={selectedNodeDisplayName}
-      onTitleChange={onTitleChange}
-    />
+          }
+          togglePanel();
+        }}
+        trackEvent={handleTrackEvent}
+        onCommentChange={onCommentChange}
+        title={selectedNodeDisplayName}
+        onTitleChange={onTitleChange}
+      />
+      <DeleteNodeModal
+        nodeId={selectedNode}
+        // nodeIcon={iconUriResult.result}
```

Comment:
If these aren't needed (commented). There is no need to commit these lines.

Reply:
These were taken from the DeleteNodeModal in ScopeCardNode. I can remove, didn't know if they were left there for context.

Reply:
They were left on the ScopeCardNode component because we were possibly going to have new modal designs that would use those props, but design never gave us anything so it was just leftover